apps/dashboard/src/components/primitives/form/avatar-picker.tsx

```diff
@@ -40,14 +40,17 @@ export const AvatarPicker = forwardRef<HTMLInputElement, AvatarPickerProps>(({ i
     <div className="space-y-2">
       <Popover modal={true} open={isOpen} onOpenChange={setIsOpen}>
         <PopoverTrigger asChild>
-          <Button variant="outline" size="icon" className="text-foreground-600 size-10">
+          <Button variant="outline" size="icon" className="text-foreground-600 relative size-10">
             {props.value ? (
-              <Avatar>
+              <Avatar className="p-[1px]">
                 <AvatarImage src={props.value as string} />
               </Avatar>
             ) : (
               <RiImageEditFill className="size-5" />
             )}
+            {error && (
+              <RiErrorWarningFill className="text-destructive outline-destructive absolute right-0 top-0 size-3 -translate-y-1/2 translate-x-1/2 rounded-full outline outline-1 outline-offset-1" />
```

Comment:
You can do `m-auto inset-0` to center this a bit cleaner i think.

Reply:
I'm not sure I get it, we want to show the error icon in the top right corner of the avatar. Can you please clarify?